cmd/commands/root.go

```diff
@@ -37,6 +37,7 @@ var (
 	KubeConfig           string
 	OperatorNamespace    string
 	CephClusterNamespace string
+	Context              string
```

Comment:
How about a more specific name?
```suggestion
	KubeContext          string
```

Reply:
done